client/src/annoMatrix/whereCache.ts

```diff
@@ -78,13 +89,16 @@ export function _whereCacheGet(
       const columnMap = whereCache?.where?.[field]?.[queryField];
       return columnMap?.get(queryColumn)?.get(queryValue) ?? [undefined];
     }
-    if (query.summarize) {
+    if ("summarize" in query) {
       const {
         method,
         field: queryField,
         column: queryColumn,
         values: queryValues,
       } = query.summarize;
+      // @ts-expect-error ts-migrate --- TODO revisit:
+      // "method" is possibly undefined (as it is an optional property on SummarizeQueryTerm) and can therefore not
```

Comment:
I'm pretty sure that `method` is required (and the type signature in query.ts needs to be updated).